src/pivot-table/hooks/use-column-width.ts

```diff
@@ -110,14 +118,14 @@ export default function useColumnWidth(
           }),
         );
       } else {
-        const { qFallbackTitle, qApprMaxGlyphCount, columnWidth } = qDimensionInfo;
+        const { title, qApprMaxGlyphCount, columnWidth } = cell;
         const iconWidth = !isFullyExpanded && index < qNoOfLeftDims - 1 ? EXPAND_ICON_WIDTH : 0;
         const fitToContentWidth = Math.max(
-          measureTextForHeader(qFallbackTitle) + TOTAL_CELL_PADDING,
-          estimateWidthForDimensionValue(qApprMaxGlyphCount) + iconWidth,
+          measureTextForHeader(title) + TOTAL_CELL_PADDING,
+          estimateWidthForDimensionValue(qApprMaxGlyphCount as number) + iconWidth,
         );
 
-        width = getColumnWidth(columnWidth, fitToContentWidth);
+        width = getColumnWidth(columnWidth as ColumnWidth, fitToContentWidth);
```

Comment:
On main `columnWidth` can be undefined. Which should be handled by `getColumnWidth`. So there should be no need to cast it here

Reply:
fixed